packages/docs/src/hooks/useIsConnected.tsx

```diff
@@ -20,9 +20,9 @@ export function useIsConnected() {
       main();
     }
 
-    fuel?.on('connection', main);
+    fuel?.on(fuel.events.CONNECTION, main);
```

Comment:
this is broken if you open the file in your local env:
![image](https://user-images.githubusercontent.com/8636507/216054952-247a18cb-5358-4fe6-bcdb-79599d59a276.png)


also the docs page `How to Use` it's broken, might be related to mentioned error
![image](https://user-images.githubusercontent.com/8636507/216055937-9e83a2cb-09fd-4456-8152-e836fc9ac183.png)